eox_nelp/pearson_vue/utils.py

```diff
@@ -40,5 +40,7 @@ def generate_client_authorization_id(user_id: int, course_id: str) -> str:
         str: string that represents the client_auth_id
     """
     course_enrollment = CourseEnrollment.objects.get(user_id=user_id, course_id=course_id)
+    anonymous_user_id = anonymous_id_for_user(course_enrollment.user, course_id)
+    anonymous_user_id_instance = AnonymousUserId.objects.get(anonymous_user_id=anonymous_user_id)
 
-    return str(course_enrollment.id + CLIENT_AUTHORIZATION_ID_OFFSET)
+    return f"{course_enrollment.id + CLIENT_AUTHORIZATION_ID_OFFSET}-{anonymous_user_id_instance.id}"
```

Comment:
I think we could remove the `CLIENT_AUTHORIZATION_ID_OFFSET`.

Reply:
I thought you like it :)